### PR TITLE
Denormalized unicode

### PR DESCRIPTION
**Pull request by igorkamyshev**
Hello! I use you lib, it's wonderful, thanks.

But, I found one minor bug, if original string contain not normalized unicode, the result string will contain artefacts.

I fix it and write test. Please, review my PR and give me feedback.

### CODE REVIEW

File: CyrillicToTranslit.js [hunk 1]
```diff
@@ -74,15 +74,17 @@ module.exports = function cyrillicToTranslit(config) {
       return "";
     }
 
+    const normalizedInput = input.normalize()
```
Comment (greybax) [inline CyrillicToTranslit.js]: please add missing `;`

Comment (greybax) [reply CyrillicToTranslit.js]: Also would be great to have a comment with some clarification before this line of code why we normalize the string and what we want to achieve.

File: test/test.js [hunk 2]
```diff
@@ -16,6 +16,9 @@ describe('cyrillicToTranslit', () => {
       assert.equal("", cyrillicToTranslit().transform());
       assert.equal("", cyrillicToTranslit().transform(null));
     });
+    it('should return translit when cyrillic string not nomalized', () => {
```
Comment (greybax) [inline test/test.js]: great! 👍

**Comment by igorkamyshev** [comment]
Thanks. Done 👌

**Pull request merged**